pkg/run/watch/forward_port.go

```diff
@@ -126,3 +134,40 @@ func ForwardPort(log logr.Logger, pod *corev1.Pod, cfg *rest.Config, specMap *Po
 
 	return fw.ForwardPorts()
 }
+
+func ShowPortForwards(log clilogger.Logger, portForwards map[string]*PortForwardShortcut) {
+	// print keyboard shortcuts
+	// print text in bold
+	fmt.Printf("\n\033[1m%s\033[0m\n\n", "We set up port forwards for you, use the number below to open it in the browser")
+
+	for key, portForward := range portForwards {
+		fmt.Printf("(%s) %s: http://localhost:%s\n", key, portForward.Name, portForward.HostPort)
+	}
+
+	fmt.Println()
+
+	// listen for keypresses
+	go func() {
```

Comment:
It is idiomatic Go to never start a goroutine without a way to shut it down. I would recommend passing in a context.Context and if that's closed, exit from this goroutine.

Reply:
I would appreciate an example.